### PR TITLE
add post about using Rmd files with new template

### PR DESCRIPTION
**Pull request by fmichonneau**
Hi @gvwilson 

Could you please review and let me know if you think it's the appropriate level of detail for the blog? Am I missing anything?

Also, it might best to wait until the new template gets merged into the R gapminder lesson before publishing this (otherwise, I'll have to edit it the links at the end of the post).

Thanks!

### CODE REVIEW

File: _posts/2016/07/2016-07-08-rmarkdown-new-template.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+authors: ["François Michonneau"]
+title: "Using RMarkdown with the new lesson template"
+date: 2016-07-08
+category: ["Tooling"]
+---
+
+Our lesson template is getting a face-lift.  Actually, it is a lot more than
+that: all the internal mechanics is also affected.
```
Comment (gvwilson) [inline _posts/2016/07/2016-07-08-rmarkdown-new-template.md]: s/is/are

File: _posts/2016/07/2016-07-08-rmarkdown-new-template.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+
+## What's new with the template?
+
+Greg has developed a [new template](http://gvwilson.github.io/lesson-example/)
```
Comment (gvwilson) [inline _posts/2016/07/2016-07-08-rmarkdown-new-template.md]: s/Greg has/The lesson maintainers have

File: _posts/2016/07/2016-07-08-rmarkdown-new-template.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+that comes with features that have been repeatedly requested such as an easy way
+to navigate among episodes within a lesson: there are now "previous" and "next"
+arrows in each episode.
+
+This new template does not require having to go through pandoc. Currently, the
```
Comment (gvwilson) [inline _posts/2016/07/2016-07-08-rmarkdown-new-template.md]: s/go through/use

File: _posts/2016/07/2016-07-08-rmarkdown-new-template.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+lessons are written in Markdown, and converted into HTML by our lesson
+maintainers using a wonderful piece of software called
+[pandoc](http://pandoc.org/).  Instead, the new template uses
+[jekyll](https://jekyllrb.com/) to take care of the conversion from Markdown
+into HTML. Lesson maintainers will still need to have jekyll installed to check
+that the website gets generated correctly, but the conversion from Markdown to
+HTML for our online lessons will be handled directly by jekyll on the GitHub
+servers. This means less work for our lesson maintainers and contributors, as
+they will only to change to the markdown files.
```
Comment (gvwilson) [inline _posts/2016/07/2016-07-08-rmarkdown-new-template.md]: It also means that we will _not_ have to put the generated HTML files in the repository, which removes another common source of error and frustration.

**Comment by gvwilson** [comment]
A few minor changes, but otherwise it looks good, and I'll fold this into the documentation once it's published - thanks!

**Comment by fmichonneau** [comment]
Thanks for the feedback! The edits are in.

**Comment by gvwilson** [comment]
Thanks @fmichonneau - @jduckles, over to you.

**Comment by jduckles** [comment]
Thanks for the post @fmichonneau! Merging.

**Pull request merged**